src/FunctionApps/python/IntakePipeline/__init__.py

```diff
@@ -0,0 +1,27 @@
+import logging
+
+import azure.functions as func
+
+from IntakePipeline.transform import transform_bundle
+from IntakePipeline.linkage import add_patient_identifier
+from IntakePipeline.fhir import read_fhir_bundles, upload_bundle_to_fhir_server
```

Comment:
Not a blocker, just for clarification: this implies that we'll move all the other function code into a single giant function within the function app. It'll be the Python function app, which has the IntakePipeline function directory, and inside of that, three sub directories, one for each of the `transform`, `linkage`, and `fhir` code. Some q's:
1. how do folks feel about having one function within the function app? I personally have no issue with it, given that our whole goal here was to create a simplified single master invocation function that called everything else. 
2. if we did move to a structure that had one function and three subdirectories (I think it'll go to 4 once we merge in Jesse's import code), does that mean we should dump the `__init__.py` and various function files for each of the subdirectories? I.e. will we have only a single `__init__.py` here in Intake Pipeline?

Reply:
On second thought, we could theoretically keep our individual functions in their own directories right now, with their own `__init__.py` files and associated tests, and then just make code copies of the things that need to be ported to work with the pipeline. Then we could just deploy the `IntakePipeline` to Azure as a one-stop shop.